### ISSUE TITLE
An ADA issue: the keyboard focus is not visible for a check-box on the frontend customer registration form

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#26]
[upwork.com/ab/f/contracts/20874295](https://www.upwork.com/ab/f/contracts/20874295)
[kitchencollection.com/customer/account/create](https://www.kitchencollection.com/customer/account/create)

**Comment by dmitry-fedyuk** [comment]
Fixed in [0.1.7](https://github.com/kitchencollection-com/core/releases/tag/0.1.7).

**Comment by dmitry-fedyuk** [comment]
![02](https://user-images.githubusercontent.com/83204/47501609-bf82b900-d887-11e8-8d73-7cb37cac252e.png)
